package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@percy/appium-app",
   "description": "Appium client library for visual testing with Percy",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Dont do version bump yet, this can go with next release

Reply:
this is for previous PR, wdio type fix

Reply:
Got it!